PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
```

Comment:
Should the file include entries for 'NSPrivacyTracking' and 'NSPrivacyTrackingDomains'?
(probably setting NSPrivacyTracking to 'false' and NSPrivacyTrackingDomains to an empty array)

Reply:
Added